<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWordListDialog</class>
 <widget class="QDialog" name="EditWordListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>512</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>768</width>
    <height>512</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#2C2E2F;
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label">
                 <property name="styleSheet">
                  <string notr="true">
    spacing: 5px;
	color: #c1c1c1;</string>
                 </property>
                 <property name="text">
                  <string>Начальное слово:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="styleSheet">
                  <string notr="true">
    spacing: 5px;
	color: #c1c1c1;</string>
                 </property>
                 <property name="text">
                  <string>Окончательные слова(через запятую):</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="styleSheet">
                  <string notr="true">
    spacing: 5px;
	color: #c1c1c1;</string>
                 </property>
                 <property name="text">
                  <string>Нужно чисел для работы:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="styleSheet">
                  <string notr="true">
    spacing: 5px;
	color: #c1c1c1;</string>
                 </property>
                 <property name="text">
                  <string>Выдает чисел после работы:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QLineEdit" name="StartWord">
                 <property name="styleSheet">
                  <string notr="true">QLineEdit
{
    border-style: solid;
    border-width: 1px;
    border-color: #27292a;
    color:#d1d1d1;
	background-color:#414243;
}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="EndWords">
                 <property name="styleSheet">
                  <string notr="true">QLineEdit
{
    border-style: solid;
    border-width: 1px;
    border-color: #27292a;
    color:#d1d1d1;
	background-color:#414243;
}</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="NeedIntegers">
                 <property name="styleSheet">
                  <string notr="true">QLineEdit
{
    border-style: solid;
    border-width: 1px;
    border-color: #27292a;
    color:#d1d1d1;
	background-color:#414243;
}</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="GivesIntegers">
                 <property name="styleSheet">
                  <string notr="true">QLineEdit
{
    border-style: solid;
    border-width: 1px;
    border-color: #27292a;
    color:#d1d1d1;
	background-color:#414243;
}</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>12</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <property name="spacing">
                  <number>4</number>
                 </property>
                 <item>
                  <widget class="QRadioButton" name="isFunction">
                   <property name="styleSheet">
                    <string notr="true">QRadioButton{
    spacing: 5px;
	color: #c1c1c1;
}
QRadioButton::indicator {
    width: 13px;
    height: 13px;
}

QRadioButton::indicator::unchecked {
    image: url(:/images/radio_button_unchecked.png);
}

QRadioButton::indicator:unchecked:hover {
    image: url(:/images/radio_button_unchecked_hover.png);
}

QRadioButton::indicator:unchecked:pressed {
    image: url(:/images/radio_button_unchecked_hover.png);
}

QRadioButton::indicator::checked {
    image: url(:/images/radio_button_checked.png);
}

QRadioButton::indicator:checked:hover {
    image: url(:/images/radio_button_checked_hover.png);
}

QRadioButton::indicator:checked:pressed {
    image: url(:/images/radio_button_checked_selected.png);
}</string>
                   </property>
                   <property name="text">
                    <string>Функция?</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="isVariable">
                   <property name="styleSheet">
                    <string notr="true">QRadioButton{
    spacing: 5px;
	color: #c1c1c1;
}
QRadioButton::indicator {
    width: 13px;
    height: 13px;
}

QRadioButton::indicator::unchecked {
    image: url(:/images/radio_button_unchecked.png);
}

QRadioButton::indicator:unchecked:hover {
    image: url(:/images/radio_button_unchecked_hover.png);
}

QRadioButton::indicator:unchecked:pressed {
    image: url(:/images/radio_button_unchecked_hover.png);
}

QRadioButton::indicator::checked {
    image: url(:/images/radio_button_checked.png);
}

QRadioButton::indicator:checked:hover {
    image: url(:/images/radio_button_checked_hover.png);
}

QRadioButton::indicator:checked:pressed {
    image: url(:/images/radio_button_checked_selected.png);
}</string>
                   </property>
                   <property name="text">
                    <string>Переменная?</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <property name="spacing">
                  <number>4</number>
                 </property>
                 <item>
                  <widget class="QCheckBox" name="one_word">
                   <property name="styleSheet">
                    <string notr="true">QCheckBox{
    spacing: 5px;
	color: #c1c1c1;
}

QCheckBox::indicator {
    width: 13px;
    height: 13px;
}

QCheckBox::indicator:unchecked {
    image: url(:/images/check-box-empty.png);
}

QCheckBox::indicator:unchecked:hover {
    image: url(:/images/check-box-empty_hover.png);
}

QCheckBox::indicator:unchecked:pressed {
    image: url(:/images/check-box-empty_hover.png);
}

QCheckBox::indicator:checked {
    image: url(:/images/check-box-checked.png);
}

QCheckBox::indicator:checked:hover {
    image: url(:/images/check-box-checked_hover.png);
}

QCheckBox::indicator:checked:pressed {
    image: url(:/images/check-box-checked_hover.png);
}
/*
QCheckBox::indicator:indeterminate:hover {
    image: url(:/images/checkbox_indeterminate_hover.png);
}

QCheckBox::indicator:indeterminate:pressed {
    image: url(:/images/checkbox_indeterminate_pressed.png);
}*/</string>
                   </property>
                   <property name="text">
                    <string>Одно слово</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="SaEWiOL">
                   <property name="styleSheet">
                    <string notr="true">QCheckBox{
    spacing: 5px;
	color: #c1c1c1;
}

QCheckBox::indicator {
    width: 13px;
    height: 13px;
}

QCheckBox::indicator:unchecked {
    image: url(:/images/check-box-empty.png);
}

QCheckBox::indicator:unchecked:hover {
    image: url(:/images/check-box-empty_hover.png);
}

QCheckBox::indicator:unchecked:pressed {
    image: url(:/images/check-box-empty_hover.png);
}

QCheckBox::indicator:checked {
    image: url(:/images/check-box-checked.png);
}

QCheckBox::indicator:checked:hover {
    image: url(:/images/check-box-checked_hover.png);
}

QCheckBox::indicator:checked:pressed {
    image: url(:/images/check-box-checked_hover.png);
}
/*
QCheckBox::indicator:indeterminate:hover {
    image: url(:/images/checkbox_indeterminate_hover.png);
}

QCheckBox::indicator:indeterminate:pressed {
    image: url(:/images/checkbox_indeterminate_pressed.png);
}*/</string>
                   </property>
                   <property name="text">
                    <string>Начальное и окончательное слово в одной строке</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="CreateNewWord">
                   <property name="styleSheet">
                    <string notr="true">QCheckBox{
    spacing: 5px;
	color: #c1c1c1;
}

QCheckBox::indicator {
    width: 13px;
    height: 13px;
}

QCheckBox::indicator:unchecked {
    image: url(:/images/check-box-empty.png);
}

QCheckBox::indicator:unchecked:hover {
    image: url(:/images/check-box-empty_hover.png);
}

QCheckBox::indicator:unchecked:pressed {
    image: url(:/images/check-box-empty_hover.png);
}

QCheckBox::indicator:checked {
    image: url(:/images/check-box-checked.png);
}

QCheckBox::indicator:checked:hover {
    image: url(:/images/check-box-checked_hover.png);
}

QCheckBox::indicator:checked:pressed {
    image: url(:/images/check-box-checked_hover.png);
}
/*
QCheckBox::indicator:indeterminate:hover {
    image: url(:/images/checkbox_indeterminate_hover.png);
}

QCheckBox::indicator:indeterminate:pressed {
    image: url(:/images/checkbox_indeterminate_pressed.png);
}*/</string>
                   </property>
                   <property name="text">
                    <string>Создает новое слово</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="styleSheet">
              <string notr="true">
    spacing: 5px;
	color: #c1c1c1;</string>
             </property>
             <property name="text">
              <string>Описание:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="desciption">
             <property name="styleSheet">
              <string notr="true">QScrollBar:horizontal {
                            color:#AFAFAF;
                        background-color:#2C2E2F;
                        }

                        QScrollBar::handle:horizontal {

                         border: 2px groove #6b6c6d;
                        border-radius:4px;
                        background-color:#2C2E2F;
                            color:#AFAFAF;
                            min-width: 40px;
                        }

                        QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
                            color:#c1c1c1;
                        background-color:#2C2E2F;
                        }

                        QScrollBar:vertical {
                            color:#AFAFAF;
                        background-color:#2C2E2F;
                         }
                         QScrollBar::handle:vertical {
                         border: 2px groove #6b6c6d;
                        border-radius:4px;
                        background-color:#2C2E2F;
                            color:#AFAFAF;
                             min-height: 80px;
                         }
                         QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
                            background: #2C2E2F;
                         }

                        QTextEdit{
                        border-style: solid;
                        border-width: 2px;
                        border-color: #27292a;
						background-color:#414243;
                        color:#FAFAFA;
                        }</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="list">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QListView
{
                        border-style: solid;
                        border-width: 2px;
                        border-color: #27292a;
						background-color:#414243;
}


QListView::item {
background-color:#4a4d4f;
 border: 2px solid #4a4d4f;
    border-bottom-color: #4a4d4f; /* same as the pane color */
    min-width: 12px;
    min-height: 12px;
    padding: 4px;
    padding-top: 2px;
	padding-bottom:02px;
}

QListView::item:selected {
    border-color: #007ACC;
    color:#FAFAFA;
	background:#007ACC;
}

QListView::item:selected:!active {
}

QListView::item:selected:active {
}

QListView::item:hover {
    background-color: #6b6c6d;
	border:#6b6c6d;
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
